app/src/panel/models/page/blueprint/pages.php

```diff
@@ -22,7 +22,7 @@ class Pages extends Obj {
   public function __construct($params = array()) {
 
     if($params === true) {
-      $this->template = blueprint::all();
+      $this->template = true;
```

Comment:
Not sure about this - also not sure if I understand you completely, could you maybe make your example more explicit? The thing is that within the blueprint logic, we always use `true` as a representation for `all`. So forwarding the `true` value from `$params` to `$this->template` and only delaying the trigger of `blueprint:all()` seemed appropriate.